src/apps/vpn/ui/screens/subscriptionNeeded/ViewSubscriptionNeeded.qml

```diff
@@ -0,0 +1,194 @@
+            Accessible.name: text.replace(/<[^>]*>/g, "")
+        }
+
+        Loader {
+            Layout.topMargin: MZTheme.theme.vSpacing
+            Layout.leftMargin: MZTheme.theme.windowMargin
+            Layout.rightMargin: MZTheme.theme.windowMargin
+            Layout.fillWidth: true
+            Layout.maximumWidth: MZTheme.theme.maxHorizontalContentWidth
+            Layout.alignment: Qt.AlignHCenter
+
+            active: vpnFlickable.isMobile
+            sourceComponent: ColumnLayout {
+                spacing: 16
+
+                ButtonGroup {
+                    id: subscriptionOptions
+                }
+
+                Repeater {
+                    id: productList
+                    model: VPNProducts
+                    delegate: MZSubscriptionOption {}
+                }
+            }
+        }
+
+
+        MZButton {
+            id: subscribeNow
+            objectName: "vpnSubscriptionNeededView"
+
+            text: MZI18n.PurchaseSubscribeNow
+
+            Layout.topMargin: MZTheme.theme.vSpacing
+            Layout.leftMargin: MZTheme.theme.windowMargin * 2
+            Layout.rightMargin: MZTheme.theme.windowMargin * 2
+            Layout.fillWidth: true
+
+            onClicked: isMobile ? VPNPurchase.subscribe(subscriptionOptions.checkedButton.productId) : VPNPurchase.subscribe("web")
+        }
+
+
+        Loader {
+            Layout.fillWidth: true
+            Layout.topMargin: MZTheme.theme.vSpacingSmall
+            Layout.alignment: Qt.AlignHCenter
+
+            active: vpnFlickable.isMobile
+            sourceComponent: RowLayout {
+
+                spacing: 0
+
+                Item {
+                    Layout.fillWidth: true
+                }
+
+                MZGreyLink {
+                    Layout.fillWidth: true
+                    Layout.maximumWidth: implicitWidth
+
+                    labelText: MZI18n.AboutUsTermsOfService
+                    onClicked: MZUrlOpener.openUrlLabel("termsOfService")
+                }
+
+                Rectangle {
+                    width: 4
+                    height: 4
+                    radius: 2
+                    opacity: .8
+                    color: MZTheme.theme.greyLink.defaultColor
+                }
+
+                MZGreyLink {
+                    Layout.fillWidth: true
+                    Layout.maximumWidth: implicitWidth
+
+                    labelText: MZI18n.AboutUsPrivacyNotice
+                    onClicked: MZUrlOpener.openUrlLabel("privacyNotice")
+                }
+
+                Item {
+                    Layout.fillWidth: true
+                }
+            }
+        }
+
+        MZLinkButton {
+            id: restorePurchase
+
+            Layout.topMargin: 8
+            Layout.alignment: Qt.AlignHCenter
+
+            visible: Qt.platform.os === "ios"
+            labelText: MZI18n.RestorePurchaseRestorePurchaseButton
+            onClicked: VPNPurchase.restore()
+        }
```

Comment:
@peterbenvenuto The "Already a subscriber" restore purchase button has always been iOS only, but the mocks show it on desktop. Is this intended for desktop as well

Reply:
Confirmed with @peterbenvenuto - "Already a subscriber" button is an iOS only feature to restore an app store purchase. Mocks have been updated